### ISSUE TITLE
Dynamically inserting links to examples

### ISSUE THREAD

**Issue by lindapaiste** [ml5js/ml5-website-v02-docsify#35]
A while back I wrote a Docsify plugin to automatically insert links to all of the examples.  I think it's really good stuff but it's just been sitting there unreviewed. 😭  https://github.com/ml5js/ml5-library/pull/1324

I'll need to make some changes to the code if the long-term plan is to keep the Docsify docs in a separate repo from the examples.  Is this something that you're interested in?

**Comment by shiffman** [comment]
@MOQN I wonder if you might have some thoughts here given the new website design and framework. Are we planning to keep the separate https://examples.ml5js.org/ page or now that we are simplifying the documentation pages with new design  is this no longer worth additionally maintaining? We no longer plan to have an `examples.json` file is that correct?

**Comment by MOQN** [comment]
Yes, that is correct. Since we aim to simplify the website development structure, we will not be maintaining `examples.json` and its page.

Our current plan is to incorporate the example codes directly within each model's reference pages. Although this might be less efficient in terms of maintainability, it will be more intuitive for both users and those new to contributing to the documentation.

We're open to discussing this further, especially since some members are interested in enhancing the collection of example pages: https://examples.ml5js.org/. If so, the page will remain on the Gatsby side with custom features. I believe this approach will still be sustainable, as the "new generation" of the library doesn't have many models. @lindapaiste, please feel free to share any insights you have. We truly appreciate your feedback! @QuinnHe @sproutleaf, feel free to add any thoughts!

**Comment by lindapaiste** [comment]
@MOQN I'm not fully understanding what the plan is.  Do you mean that you will be deleting this section from the doc? https://github.com/ml5js/ml5-website-v02-docsify/blob/main/docs/reference/neural-network.md#examples  How do the examples in https://github.com/ml5js/ml5-next-gen/tree/main/examples get incorporated into the website and/or documentation?

**Comment by MOQN** [comment]
Hi @lindapaiste! We will not delete the example sections from the document. However, we will keep only those examples that link to the p5.js web editor, to minimize confusion for beginners in coding.

As mentioned, the example codes in https://github.com/ml5js/ml5-next-gen/tree/main/examples will be incorporated into each model’s documentation pages. The Docsify plugin you've developed may be integrated into the individual pages to describe the example codes more effectively.

The ongoing discussion about the example page is about making the reference page more intuitive and engaging with custom features on the Gatsby platform. We are currently in the ideation stage, planning a new design/interface based on this idea: https://examples.ml5js.org/.
